js/package.json

```diff
@@ -7,8 +7,8 @@
 		"preview": "svelte-kit preview",
 		"check": "svelte-check --tsconfig ./tsconfig.json",
 		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
-		"lint": "prettier --check --plugin-search-dir=. .",
-		"format": "prettier --write --plugin-search-dir=. .",
+		"lint": "prettier --check src && eslint --ext .js,.mjs,.ts,.svelte src",
```

Comment:
could be nice to align the command names on the `moon-landing` package as much as we can ?
```
    "lint": "eslint --quiet --fix --ext .js,.mjs,.ts,.svelte",
    "lint:all": "npm run lint .",
    "format:all": "prettier --write . && npm run lint:all",
    "prettier": "prettier --write",
```